msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -237,7 +237,7 @@ Anonymous users can view and interact with Adaptive Cards in the meeting chat. A
 
 |Sample name | Description | .NET |
 |----------------|-----------------|--------------|--------------|
-| Anonymous user support | Sample app to show anonymous user support in meeting apps. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-anonymous-users/csharp) |
+| Anonymous user support | Sample app to show anonymous user support in meeting apps (Side panel and Meeting stage). | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-anonymous-users/csharp) |
```

Comment:
Sample app to show anonymous user support in `Teams` meeting apps (Side panel and Meeting stage). It also showcases Azure AD and Facebook authentications.

Reply:
Changed